src/containers/AdminCampaignEdit.jsx

```diff
@@ -1,5 +1,6 @@
 import PropTypes from 'prop-types'
 import React from 'react'
+import camelCase from 'camelcase'
```

Comment:
i think we can do this with lodash (already on the project)

Reply:
https://lodash.com/docs/4.17.4#camelCase

Reply:
Ah lodash, is there anything it can't do?